lib/contract/ind.js

```diff
@@ -2,14 +2,13 @@ var assert = require('assert');
 
 var _ = require('lodash');
 
-module.exports = function (symbol, expiry, currency, exchange) {
+module.exports = function (symbol, currency, exchange) {
   assert(_.isString(symbol), 'Symbol must be a string.');
   assert(_.isString(expiry), 'Expiry must be a string.');
```

Comment:
@athena123 this assertion will throw now that expiry has been removed.

Reply:
you are right. thanks.